packages/dds/merge-tree/src/test/client.localReferenceFarm.spec.ts

```diff
@@ -22,8 +22,8 @@ import { TestClientLogger } from "./testClientLogger";
 
 const defaultOptions: Record<"initLen" | "modLen", IConfigRange> & IMergeTreeOperationRunnerConfig =
 	{
-		initLen: { min: 2, max: 4 },
-		modLen: { min: 1, max: 8 },
+		initLen: { min: 2, max: 256 },
+		modLen: { min: 1, max: 256 },
```

Comment:
See commit message in https://github.com/microsoft/FluidFramework/pull/17675/commits/d4ebf222cec53215ecf4ce781e5e11d8c87cc541

Reply:
neat!